devel/libical/files/patch-nodocs.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 23898b6..fb7e452 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -347,7 +347,7 @@ endif()
+ ################# build subdirs ########################
+ 
+ add_subdirectory(design-data)
+-add_subdirectory(doc)
++# add_subdirectory(doc)
```

Comment:
I would go for capital letters, but that's a matter of personal preference

```
-DBUILD_DOC[S]=FALSE
```

Again, there's nothing wrong if you don't change this, it's just a tiny bit cleaner to use a configure option and install a patch unconditionally (or even better to convince upstream to include it).